src/main/kotlin/org/dropProject/controllers/ReportController.kt

```diff
@@ -95,22 +99,63 @@ class ReportController(
 
     val LOG = LoggerFactory.getLogger(this.javaClass.name)
 
+    /**
+     * Controller that handles requests for the list of signalled groups in an [Assignment].
+     * The signalled groups are groups of students that are failing exactly the same tests.
+     * @param assignmentId is a String identifying the relevant Assignment
+     * @param model is a [ModelMap] that will be populated with information to use in a View
+     * @param principal is a [Principal] representing the user making the request
+     * @param request is an [HttpServletRequest]
+     * @return A String with the name of the relevant View
+     */
+    @RequestMapping(value = ["/cenas/{assignmentId}"], method = [(RequestMethod.GET)])
+    fun getSignaledGroupsOrSubmissions(@PathVariable assignmentId: String, model: ModelMap,
+    principal: Principal, request: HttpServletRequest): String {
+        model["assignmentId"] = assignmentId
+
+        assignmentService.getAllSubmissionsForAssignment(assignmentId, principal, model, request,
+                includeTestDetails = true,
+                mode = "signalledSubmissions")
+
+        return "signalledSubmissions"
```

Comment:
I know I'm being nit-picky but can you change this template's name to "signalled-submissions", to be coherent with the naming scheme used by the other templates?

Reply:
Done.